starters/default/gatsby-config.js

```diff
@@ -3,6 +3,9 @@ module.exports = {
     title: `Gatsby Default Starter`,
     description: `Kick off your next, great Gatsby project with this default starter. This barebones starter ships with the main Gatsby configuration files you might need.`,
     author: `@gatsbyjs`,
+    keywords: [`gatsby`, `application`, `react`],
+    image: `https://www.dropbox.com/s/bhwnfkuwewq03sl/gatsby-share.png?raw=1`,
```

Comment:
Image should be in GitHub. Not at dropbox

Reply:
@muescha done, just tested with https://raw.githubusercontent.com/gatsbyjs/gatsby/master/starters/default/src/images/gatsby-astronaut.png

After merge, the new raw image link (https://raw.githubusercontent.com/gatsbyjs/gatsby/master/starters/default/src/images/gatsby-share.png) will be pulled from the official gatsby repo.